### PR TITLE
[quickstarts] Small updates and cross-linking

### PR DESCRIPTION
**Pull request by hhunter-ms**
Signed-off-by: Hannah Hunter <hannahhunter@microsoft.com>

## Description

- Update descriptions on quickstarts/_index.md
- Update C# and Go code snippets for Bindings
- Link to quickstarts from Service Invo and Secrets how-tos

## Issue reference

#2573

### CODE REVIEW

File: daprdocs/content/en/getting-started/quickstarts/_index.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ Hit the ground running with our Dapr quickstarts, complete with code samples aim
 
 | Quickstarts | Description |
 | ----------- | ----------- |
-| [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) |  Call a method on another service using the service invocation API. |
-| [State Management]({{< ref statemanagement-quickstart.md >}}) | Create stateful applications using the state management API. |
-| [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Send and receive messages using the publish and subscribe API. |
-| [Bindings]({{< ref bindings-quickstart.md >}}) | Schedule a database insert job using the input and output bindings API. |
-| [Secrets Management]({{< ref secrets-quickstart.md >}}) | Retrieve secrets in the application code from a configured secret store using the secrets management API. |
+| [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) |  Asynchronous communication between two services using messaging. |
```
Comment (paulyuk) [inline daprdocs/content/en/getting-started/quickstarts/_index.md:25]: Like!

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 2]
```diff
@@ -99,23 +99,11 @@ def process_batch():
...
-        # Insert order using Dapr output binding via HTTP Post
-        resp = d.invoke_binding(binding_name=sql_binding, operation='exec',
-                                binding_metadata=payload, data='')
-        return resp
-    except Exception as e:
-        print(e, flush=True)
-        raise SystemExit(e)
+try:
+    # Insert order using Dapr output binding via HTTP Post
+    resp = d.invoke_binding(binding_name=sql_binding, operation='exec',
```
Comment (paulyuk) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md:113]: I like it that you've shortened.  however I think the line with `With DaprClient as d:` is still needed to make sense of the `d.invoke_binding` line.  Also for bindings the metadata/payload is vital, so I think you need all the lines from d's declaration to the invoke_binding, inclusive of the SQL command and the payload..

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 3]
```diff
@@ -905,24 +899,21 @@ dapr run --app-id batch-sdk --app-port 6002 --dapr-http-port 3502 --dapr-grpc-po
...
-	Operation: "exec",
-	Data:      []byte(""),
-	Metadata:  map[string]string{"sql": sqlCmd},
-}
+	// Insert order using Dapr output binding via Dapr SDK
+	in := &dapr.InvokeBindingRequest{
+		Name:      sqlBindingName,
+		Operation: "exec",
+		Data:      []byte(""),
+		Metadata:  map[string]string{"sql": sqlCmd},
```
Comment (paulyuk) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md:936]: Per above I think you might need to show dapr being declared as well as the sqlcmd.

**Review by paulyuk — CHANGES_REQUESTED** [review]
Hi - like the spirit of the changes.  Might have trunc'ed a little too much of the code.  but like how it's simplified!

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 4]
```diff
@@ -499,22 +496,28 @@ dapr run --app-id batch-sdk --app-port 7002 --components-path ../../../component
...
-string jsonFile = File.ReadAllText("../../orders.json");
-var ordersArray = JsonSerializer.Deserialize<Orders>(jsonFile);
-```
-
-The `batch-sdk` service uses the PostgreSQL output binding defined in the [`binding-postgres.yaml`]({{< ref "#componentbinding-postgresyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. 
+app.MapPost("/" + cronBindingName, async () => {
 
-```csharp
-using var client = new DaprClientBuilder().Build();
+    Console.WriteLine("Processing batch..");
```
Comment (paulyuk) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]: I think in this small section we're trying to explain the cron binding, so I would remove all this file processing stuff.

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 5]
```diff
@@ -499,22 +496,28 @@ dapr run --app-id batch-sdk --app-port 7002 --components-path ../../../component
...
         Console.WriteLine(sqlText);
+    }
+});
+```
+
+The `batch-sdk` service uses the PostgreSQL output binding defined in the [`binding-postgres.yaml`]({{< ref "#componentbinding-postgresyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table. 
+
+```csharp
+// Insert order using Dapr output binding via Dapr Client SDK
+await client.InvokeBindingAsync(bindingName: sqlBindingName, operation: "exec", data: "", metadata: command);
```
Comment (paulyuk) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md:521]: Right above HERE is where I suggest showing how the Client object is created, the sql command and the metadata command gets set.  This makes full sense of calling the output binding on line 520

Comment (paulyuk) [reply daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md:521]: What you did for Go below looks right.  I would replicate for C# equiv

**Review by paulyuk — CHANGES_REQUESTED** [review]

**Review by paulyuk — APPROVED** [review]
LGTM!

**Review by greenie-msft — APPROVED** [review]

**Pull request merged**